app/scripts/modules/core/src/managed/ManagedResourceStatusIndicator.tsx

```diff
@@ -27,11 +28,8 @@ const viewConfigurationByStatus: { [status in ManagedResourceStatus]: IViewConfi
           <b>Action is being taken to resolve a drift from the declarative configuration.</b>
         </p>
         <p>
-          Check the{' '}
-          <UISref to="home.applications.application.tasks">
-            <a>tasks view</a>
-          </UISref>{' '}
-          to see work that's in progress. <LearnMoreLink resourceSummary={resourceSummary} />
+          Check this resource's History to see details and track the work currently in progress.{' '}
```

Comment:
Newbie question: is the `{' '}` the "right" way of forcing a space character?

Reply:
yep! it's a sort of weird pattern, but this is how you make sure you end up with a space between the end of a text block and the next JSX element _if the point where they meet is broken across a line break_. If it's all on one line a normal space is enough (e.g. `this is some text <MyComponent />`. If you write a single-line thing like my example and then run it through Prettier, it will automatically add that syntax if it decides to break your JSX component onto the next line (that's what happened here).